mysql/datadog_checks/mysql/mysql.py

```diff
@@ -348,6 +345,9 @@ def _collect_metrics(self, db, tags):
             self.log.debug("Collecting Galera Metrics.")
             metrics.update(GALERA_VARS)
 
+        if self.performance_schema_enabled:
+            self._collect_users_connected_metric(db)
```

Comment:
do we care that this would mean this metric is no longer a DBM specific thing - it falls under the guise of the "activity product" offering we provide

Reply:
I believe it's okay. The only attribute that's gated behind DBM for the equivalent metric in Postgres and SQL Server is the extra tags, but since we've decided to do this in the integration for MySQL, I think we'll follow up and do the same for Postgres and SQL Server.